twitter-streams/docker-compose.yml

```diff
@@ -0,0 +1,227 @@
+version: "2.1"
+services:
+  # Begin zookeeper section
+  zookeeper:
+    image: confluentinc/cp-zookeeper:5.0.1
+    restart: always
+    hostname: zookeeper
+    environment:
+      ZOOKEEPER_SERVER_ID: 1
+      ZOOKEEPER_CLIENT_PORT: "2181"
+      ZOOKEEPER_TICK_TIME: "2000"
+      ZOOKEEPER_SERVERS: "zookeeper:2888:3888"
+    ports:
+      - "2181:2181"
+  # End zookeeper section
+
+  # Begin kafka1 section
+  kafka1:
+    image: confluentinc/cp-enterprise-kafka:5.0.1
+    hostname: kafka1
+    depends_on:
+      - zookeeper
+    # persisting data in local fs
+    volumes:
+      - $PWD/data/kafka1/data:/var/lib/kafka/data
+    ports:
+      - "9091:9091"
+      - "29091:29091"
+    environment:
+      KAFKA_ZOOKEEPER_CONNECT: "zookeeper:2181"
+      KAFKA_LISTENER_SECURITY_PROTOCOL_MAP: PLAINTEXT:PLAINTEXT,PLAINTEXT_HOST:PLAINTEXT
+      KAFKA_INTER_BROKER_LISTENER_NAME: PLAINTEXT
+      KAFKA_ADVERTISED_LISTENERS: PLAINTEXT://kafka1:9091,PLAINTEXT_HOST://localhost:29091
+      KAFKA_LOG4J_LOGGERS: "kafka.authorizer.logger=INFO"
+      KAFKA_METRIC_REPORTERS: "io.confluent.metrics.reporter.ConfluentMetricsReporter"
+      KAFKA_BROKER_ID: 1
+      KAFKA_BROKER_RACK: "r1"
+      KAFKA_OFFSETS_TOPIC_REPLICATION_FACTOR: 2
+      CONFLUENT_METRICS_REPORTER_BOOTSTRAP_SERVERS: "kafka1:9091"
+      CONFLUENT_METRICS_REPORTER_TOPIC_REPLICAS: 2
+      CONFLUENT_METRICS_REPORTER_MAX_REQUEST_SIZE: 10485760
+      # To avoid race condition with control-center
+      CONFLUENT_METRICS_REPORTER_TOPIC_CREATE: "false"
+      KAFKA_DELETE_TOPIC_ENABLE: "true"
+      KAFKA_AUTO_CREATE_TOPICS_ENABLE: "false"
+      KAFKA_LOG4J_ROOT_LOGLEVEL: INFO
+      KAFKA_JMX_PORT: 9991
+  # End kafka1 section
+
+  # Begin kafka2 section
+  kafka2:
+    image: confluentinc/cp-enterprise-kafka:5.0.1
+    hostname: kafka2
+    depends_on:
+      - zookeeper
+    volumes:
+      - $PWD/data/kafka2/data:/var/lib/kafka/data
+    ports:
+      - "9092:9092"
+      - "29092:29092"
+    environment:
+      KAFKA_ZOOKEEPER_CONNECT: "zookeeper:2181"
+      KAFKA_LISTENER_SECURITY_PROTOCOL_MAP: PLAINTEXT:PLAINTEXT,PLAINTEXT_HOST:PLAINTEXT
+      KAFKA_INTER_BROKER_LISTENER_NAME: PLAINTEXT
+      KAFKA_ADVERTISED_LISTENERS: PLAINTEXT://kafka2:9092,PLAINTEXT_HOST://localhost:29092
```

Comment:
FWIW I put this in all my Docker Compose files to help people copy & pasting and debugging their own setups: https://github.com/confluentinc/demo-scene/blob/master/mysql-debezium-ksql-elasticsearch/docker-compose/docker-compose.yml#L15-L17 /

Reply:
👍